src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -795,6 +795,7 @@ OSXOutput::Open(AudioFormat &audio_format)
 	PcmExport::Params params;
 	params.alsa_channel_order = true;
 	bool dop = dop_setting;
+	params.dsd_mode = PcmExport::DsdMode::NONE;
```

Comment:
This is redundant. It's the default-initialized value.

Reply:
Done